modules/core/src/lib/animation-loop.js

```diff
@@ -91,6 +91,11 @@ export default class AnimationLoop {
     this._onMouseleave = this._onMouseleave.bind(this);
   }
 
+  finalize() {
```

Comment:
Who would be calling this?

Reply:
> Can you provide an example of usage? I'm not totally clear on how the application would use these methods.

The website will call `animationLoop._setDisplay(new VRAnimationDisplay())` to VR enable examples.

> Who would be calling `finalize`?

E.g. deck.gl should call `finalize` on all the resources it creates in its own 'finalize` method.

Reply:
I'll change to delete() to match the rest of luma